src/qml/qgismobileapp.qml

```diff
@@ -2236,8 +2243,37 @@ ApplicationWindow {
       }
 
       Connections {
+        id: mapSettingsConnections
```

Comment:
We don't need that anymore with other proposed changes.

FYI, I think we should avoid adding functions within connections, that should only contain signals captured, otherwise it becomes harder to read.